if not os.path.exists(path):

```diff
@@ -816,7 +816,7 @@ def map_filename(filename):
 
 class WLSH_Image_Save_With_Prompt_File:
     def __init__(self):
-        self.output_dir = os.path.join(os.getcwd()+'/ComfyUI', "output")
+        self.output_dir = os.path.join(os.getcwd()+'\ComfyUI', "output")
```

Comment:
This is only correct for Windows systems.  The `/` for path delimiters should work on any system as Python should re-interpret.